docs/network/ovn_kubernetes_traffic_flows.md

```diff
@@ -0,0 +1,445 @@
+# OVNKubernetes Traffic Flows in MicroShift
+
+## OVN Topology
+
+The following diagram captures the OVN network topology for the MicroShift deployment:
+
+![picture_alt](ovn_topology.svg "OVN Topology")
+
+Below is the node/service/pod information used in the traffic flow examples:
+
+```text
+(host)$ oc get nodes -o wide
+NAME                                  STATUS   ROLES                         AGE    VERSION   INTERNAL-IP      EXTERNAL-IP   OS-IMAGE
+openshift.default.svc.cluster.local   Ready    control-plane,master,worker   6d6h   v1.25.0   192.168.122.14   <none>        Red Hat Enterprise Linux 8.6 (Ootpa)
+
+(host)$ oc get pods -A -o wide
+NAMESPACE                  NAME                                  READY   STATUS       RESTARTS     AGE    IP               NODE
+default                    pod-1                                 1/1     Running      0            3d     10.42.0.12       openshift.default.svc.cluster.local
+default                    pod-2                                 1/1     Running      0            3d     10.42.0.13       openshift.default.svc.cluster.local
+openshift-dns              dns-default-pt2vr                     2/2     Running      0            6d6h   10.42.0.7        openshift.default.svc.cluster.local
+openshift-dns              node-resolver-87kvk                   1/1     Running      0            6d6h   192.168.122.14   openshift.default.svc.cluster.local
+openshift-ingress          router-default-ddc545d88-9bfcb        1/1     Running      0            6d6h   10.42.0.5        openshift.default.svc.cluster.local
+openshift-ovn-kubernetes   ovnkube-master-jlg4f                  4/4     Running      0            6d6h   192.168.122.14   openshift.default.svc.cluster.local
+openshift-ovn-kubernetes   ovnkube-node-jkqrl                    1/1     Running      0            6d6h   192.168.122.14   openshift.default.svc.cluster.local
+openshift-service-ca       service-ca-77fc4cc659-5rzr4           1/1     Running      0            6d6h   10.42.0.4        openshift.default.svc.cluster.local
+[...]
+
+(host)$ oc get svc -A
+NAMESPACE           NAME                      TYPE        CLUSTER-IP      EXTERNAL-IP   PORT(S)                   AGE
+default             kubernetes                ClusterIP   10.43.0.1       <none>        443/TCP                   6d6h
+default             svc-nodeport              NodePort    10.43.19.182    <none>        7000:30700/TCP            30m
+openshift-dns       dns-default               ClusterIP   10.43.0.10      <none>        53/UDP,53/TCP,9154/TCP    6d6h
+openshift-ingress   router-internal-default   ClusterIP   10.43.179.188   <none>        80/TCP,443/TCP,1936/TCP   6d6h
+```
+
+## Pod Network
+
+- All the pods (non hostnetwork) are connected to an OVN logical switch named after the node name.
```

Comment:
That is a good question. 
Ovnk builds its database based on k8s API data, in case the database is lost, it can recover from the API data saved in etcd. So when the node name changed, I presume ovnk will rebuild the logcial networks with the updated node name. However, I didn't try if the rebuilt logical network will work as expected or not in microshift.